conjure-codegen/src/example_types/product/alias_as_map_key_example.rs

```diff
@@ -18,6 +27,11 @@ pub struct AliasAsMapKeyExample {
         default,
         map(key(type = super::RidAliasExample), value(type = super::ManyFieldExample))
     )]
+    #[serde(
+        rename = "rids",
+        skip_serializing_if = "std :: collections :: BTreeMap :: is_empty",
```

Comment:
are the spaces here ok? intentional?

Reply:
(not mentioned in https://serde.rs/field-attrs.html)

Reply:
It works, but is just a big ugly. I'll fix.

Reply:
yeah guess `quote!` just puts them in and they don't hurt

Reply:
Fixed!